src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int GROW_FACTOR = 2;
+
+    private Node<K,V>[] nodes;
```

Comment:
```suggestion
    private Node<K, V>[] nodes;
```
autoformat this file